caracal/schema/transform_schema.yml

```diff
@@ -15,25 +15,25 @@ mapping:
         required: false
         example: 'target'
       label_in:
-        desc: Label of the input dataset. If split_field/changecentre below are enabled, label_in must be a single value. Else, if concat below is enabled, label_in must include at least two comma-separated values.
+        desc: Label of the input dataset.
```

Comment:
@francescaLoi @o-smirnov I think this change in the schema is not correct. It is still the case (also in the polcal branch) that this worker can be run in a concat mode, in which case the label_in must be a list of comma-separated values. Can you please double check and, if I'm right, revert to the previous description of label_in?

Reply:
I agree with you. I'll change the `label_in` description.